automation/taskcluster/lib/tasks.py

```diff
@@ -341,27 +341,25 @@ def _craft_default_task_definition(
             },
         }
 
-    def craft_master_commit_signing_task(
+    def craft_raptor_signing_task(
         self, assemble_task_id, variant
     ):
-        architecture, build_type = get_architecture_and_build_type_from_variant(variant)
-        build_type = convert_camel_case_into_kebab_case(build_type)
-
+        architecture, _ = get_architecture_and_build_type_from_variant(variant)
         routes = []
         if self.repo_url == _OFFICIAL_REPO_URL:
             routes = [
                 'index.project.mobile.fenix.v2.branch.master.revision.{}.{}.{}'.format(
-                    self.commit, build_type, architecture
+                    self.commit, 'raptor', architecture
```

Comment:
Okay to land these breaking changes because appending `.raptor` to the package name already breaks the harness. Thank you, @mitchhentges, for telling me this offline